cli/tests/e2e/snapshots/test_semgrep_core_parse_error/test_rule_parser__failure__error_messages/settings0/out.json

```diff
@@ -19,12 +19,12 @@
         },
         {
           "end": {
-            "col": 12,
+            "col": 14,
```

Comment:
Why the column number changed here?

Reply:
This was from a change in tree-sitter error recovery. See also https://returntocorp.slack.com/archives/C01NXGX2EHZ/p1660624674796879